plugins/org.jkiss.dbeaver.ext.mysql.ui/src/org/jkiss/dbeaver/ext/mysql/ui/views/MySQLConnectionPage.java

```diff
@@ -194,7 +195,8 @@ public void widgetSelected(SelectionEvent e) {
             serverTimezoneCombo.setLayoutData(new GridData(GridData.HORIZONTAL_ALIGN_BEGINNING));
         }
 
-        if (DBWorkbench.hasFeature(DBConnectionConstants.PRODUCT_FEATURE_ADVANCED_DATABASE_ADMINISTRATION)) {
+        boolean supportsClients = CommonUtils.getBoolean(driver.getDriverParameter("supportsClients"), true);
```

Comment:
Maybe introduce a constant for this string?

Reply:
Just for two places? :(